test/integration/accounts.js

```diff
@@ -27,6 +27,7 @@ describe('(/accounts) Accounts', () => {
   const responseKeys = [
     'id', 'enterprise_id', 'locked', 'created', 'reference_id',
     'number', 'label', 'parent', 'type_id', 'type', 'translation_key',
+    'cost_center_id',
```

Comment:
Can you add a test where you update a cost center to the integration tests?  Just for completeness.